src/main/java/io/github/cottonmc/cotton/gui/widget/WTiledSprite.java

```diff
@@ -0,0 +1,55 @@
+package io.github.cottonmc.cotton.gui.widget;
+
+import io.github.cottonmc.cotton.gui.client.ScreenDrawing;
+import net.minecraft.util.Identifier;
+
+// It's a Sprite that can be tiled, but does not support animations
+public class WTiledSprite extends WSprite {
+  public int tileWidth = 20;
+  public int tileHeight = 20;
```

Comment:
```suggestion
	private int tileWidth;
	private int tileHeight;
```
These should be private and have no default values.

Reply:
done in a6c26d0bdc71747cdba2f95626a1f2ba8f91db89